### DISCUSSION TITLE
Kiwi browser

### DISCUSSION THREAD

**Discussion by Jokopentil** [realodix/AdBlockID#1807] [Q&A] [answered]
Mengapa lebih di sarankan pakai browser berbasis Firefox 
Di banding Chromium???
![Screenshot_2024-07-06-17-01-01-52_320a9a695de7cdce83ed5281148d6f19.jpg](https://github.com/realodix/AdBlockID/assets/114223791/46150be8-e727-4043-b085-3d460f7930b0)

**Comment by realodix** [comment, marked as answer]
> Mengapa lebih di sarankan pakai browser berbasis Firefox
> Di banding Chromium???

Ini untuk mobile ya? Dari SS Saya lihat itu mengarah ke mobile. 

Alasan utamanya hanya untuk mengurangi laporan kesalahpahaman saja. Disebut laporan kesalahpahaman karena masalah sebenarnya bukan ada di AdBlockID.

Saat ini yang Saya tahu browser mobile berbasis Chromium hanya ada Kiwi yang bisa install extensi adblocker. Sebelumnya Saya pernah menyarankan Kiwi, namun ntah mengapa belakangan ini Saya sering menemukan masalah yang hanya terjadi di Kiwi. Akhirnya Saya memutuskan untuk mengganti Kiwi dengan Waterfox di https://github.com/realodix/AdBlockID/commit/7d29d5021f30e2b5a0dfcd78e532b0afd68eec17

Mungkin Anda memiliki browser alternatif lainnya?

**Reply by Jokopentil** [reply]
Saya punya alternatif lemur browser dan yandex browser 
Tolong untuk admin untuk di uji coba 
Soalnya lemur basicnya adalah edge
Request 
Sebelum saya total menggunakan Firefox 🙏

**Reply by Jokopentil** [reply]
Untuk yandex adblock yang bisa di install sementara cuma adguard beta dan ublock origin dev build

**Reply by 3xploiton3** [reply]
sampai saat ini saya masih menggunakan kiwi + UBO (filter adblockID), 
sejauh pemakaian blm ada masalah yg critical

**Reply by doctorreditshere** [reply]
> sampai saat ini saya masih menggunakan kiwi + UBO (filter adblockID), sejauh pemakaian blm ada masalah yg critical

Bahaya bang soal keamanan masih pakai kiwi browser , udah lama ga di update sama developer.
Alternatif nya, bisa edge menggunakan ekstensi tapi punya 5 Tracker , atau lemur dan quoetta ( aku sih ga percaya keduanya ) 
 paling terbaik kalo untuk privasi itu ironfox ( fork dari mull- firefox ) kalo tetep mau bertahan di chorium , bisa pakai cromite dia juga cocok untuk yang peduli privasi

**Reply by Jokopentil** [reply]
> > sampai saat ini saya masih menggunakan kiwi + UBO (filter adblockID), sejauh pemakaian blm ada masalah yg critical
> 
> Bahaya bang soal keamanan masih pakai kiwi browser , udah lama ga di update sama developer.
> Alternatif nya, bisa edge menggunakan ekstensi tapi punya 5 Tracker , atau lemur dan quoetta ( aku sih ga percaya keduanya ) 
>  paling terbaik kalo untuk privasi itu ironfox ( fork dari mull- firefox ) kalo tetep mau bertahan di chorium , bisa pakai cromite dia juga cocok untuk yang peduli privasi

Mungkin maksudmu bromite
Masalahnya itu filter untuk regional Indonesia kayaknya gak ada

**Reply by Jokopentil** [reply]
Terlalu ribet pakai bromite jika tak faham filter adblocknya

**Reply by doctorreditshere** [reply]
> > > sampai saat ini saya masih menggunakan kiwi + UBO (filter adblockID), sejauh pemakaian blm ada masalah yg kritis
> > 
> > 
> > Bahaya bang soal keamanan masih pakai kiwi browser , udah lama ga di update sama developer. 
> > Alternatif nya, bisa edge menggunakan ekstensi tapi punya 5 Tracker , atau lemur dan quoetta ( aku sih ga percaya keduanya ) 
> > paling terbaik kalo untuk privasi itu ironfox ( fork dari mull- firefox ) kalo tetep mau bertahan di chorium , bisa pakai cromite dia juga cocok untuk yang peduli privasi
> 
> Mungkin maksudmu bromite masalah itu filter untuk regional Indonesia kayaknya gak ada

Bromite udah ke di tinggal dev nya bang, , sekarang ada pengganti nya nama nya cromite ,, fork dari bromite, ngga ribet juga kok adblock nya.

**Reply by Jokopentil** [reply]
Bromite dan cromite itu format filternya sama
Lagian filter Indonesia gak ada yg bikin 
Tak seperti adblock extension bisa blokir pop up dan video karena filternya kalah lengkap di bandingkan adblock versi extension

**Reply by doctorreditshere** [reply]
> Bromite dan cromite itu format filternya sama Lagian filter Indonesia gak ada yg bikin Tak seperti adblock extension bisa memblokir pop up dan video karena filternya kalah lengkap di bandingkan adblock versi extension

Cobain dulu bang sebelum koment, walau filter keduanya agak mirip, cromite ada tools tambahan dari dev nya...aku juga Cobain filter indonesia bisa banget disana.
Soal rencana ekstensi , dev juga lagi merencanakan nya
https://github.com/uazo/cromite/issues/256#issuecomment-2589295853

**Reply by Jokopentil** [reply]
Kiwi browser GK di perbarui karena masih pakai mv2 kemarin sudah saya coba adguard terbaru malah GK berfungsi lalu saya pasang yg versi mv2 ternyata bisa dan ini berlaku juga untuk yandex atau browser yang lain seperti quetta dan yandex browser

**Reply by doctorreditshere** [reply]
> Kiwi browser GK di perbarui karena masih pakai mv2 kemarin sudah saya coba adguard terbaru malah GK berfungsi lalu saya pasang yg versi mv2 ternyata bisa dan ini berlaku juga untuk yandex atau browser yang lain seperti quetta dan yandex browser

Kalo soal mv3 , semua chorium kena dampak nya , karena google itu yang punya chrome dan sponsor terbesar chorium itu chrome, dan ga lama lagi berdampak juga buat browser kek quetta (klo dia peduli ) , atau apapun browser yang masih peduli dengan mv2..
Sayangnya cuma kiwi yang memang project nya open source dan yang peduli dengan mv2 ,,, browser yg basisnya chromium mv3 mah ga ngurus mau dia donwload apa...sedangkan kebanyakan orang itu pakai browser android yg punya ekstensi , hal yang pertama mereka mau lakuin itu donwload ublock origin/ adguard.
Hal lainnya juga firefox mulai mendukung mv3 karena semua orang tau firefox juga dapat pembiayaan dari google ( karena mereka dari mozilla yang non profit ). Tapi mereka berjanji ga bakal ninggalin mv2 di blognya.
Berikut fork dari firefox :
Bisa buat rekomendasi @realodix 
1. Ironfox ( fork dari mull - firefox )
2. Cookiejarbrowser preview
3. Waterfox
4. Fennec fdroid

**Reply by realodix** [reply]
@doctorreditshere Terima kasih untuk sarannya. Pas ada waktu, Saya akan mencobanya satu per satu.

**Reply by Jokopentil** [reply]
Untuk ublock origin versi Chromium kayaknya belum ada yg versi mv3 
Ada tapi versi lite kalah cepat update dengan adguard 
Sementara masih nyaman pakai kiwi dan ublock origin tapi akan saya coba recommended nya

**Comment by realodix** [comment]
Saya sudah menambahkan Lemur Browser sebagai pilihan alternatif di https://github.com/realodix/AdBlockID/commit/f56e9be24bfab8b9f45cb56d7392842732a14558

---

> Untuk yandex adblock yang bisa di install sementara cuma adguard beta dan ublock origin dev build
> https://github.com/realodix/AdBlockID/discussions/1807#discussioncomment-9976839

> Microsoft Edge for Android now supports installing ANY extension... with some work
> https://www.windowscentral.com/software-apps/browsing/microsoft-edge-for-android-now-supports-installing-any-extension-with-some-work

Kedepan, saat Yandex dan Edge bisa install extensi adblocker tanpa ribet, menurut Saya mereka bisa menjadi koleksi pilihan yang baik untuk menggantikan Lemur dan Waterfox.
